index.html

```diff
@@ -302,16 +305,15 @@
 
 			function updateBlob() {
 
-				blob.reset();
-
-				for ( var i = 0; i < points.length; i ++ ) {
+				for ( var i = lastPointAddedToBlob; i < points.length; i ++ ) {
```

Comment:
Interesting. Do you mind doing a different Pull Request only with this improvement?